core/src/test/scala/it/parttimeteam/core/prolog/PrologGameSuite.scala

```diff
@@ -0,0 +1,41 @@
+package it.parttimeteam.core.prolog
+
+import it.parttimeteam.core.cards.Card
+import org.scalatest.funsuite.AnyFunSuite
+
+class PrologGameSuite extends AnyFunSuite {
```

Comment:
This test class can be reduced to only one property test class? (The PrologGamePropSpec class)?